data/api/apiBlock.go

```diff
@@ -38,6 +38,20 @@ type Block struct {
 	EpochStartInfo         *EpochStartInfo        `json:"epochStartInfo,omitempty"`
 	EpochStartShardsData   []*EpochStartShardData `json:"epochStartShardsData,omitempty"`
 	ScheduledData          *ScheduledData         `json:"scheduledData,omitempty"`
+	PreviousHeaderProof    *HeaderProof           `json:"previousHeaderProof,omitempty"`
+	Proof                  *HeaderProof           `json:"proof,omitempty"`
```

Comment:
how current proof will be used? when it will be set?

Reply:
The current proof will be set on the indexer block